nab/labeler.py

```diff
@@ -188,15 +192,14 @@ def write(self, destPath):
 
 
   def combine(self):
-    """Combine UserLabels."""
+    """Combine the raw user labels, and set the relaxed anomaly windows."""
     self.getUserLabels()
-    self.combineLabels()
-    self.combineWindows()
+    self.combineRawLabels()
     self.relaxWindows()
 
 
   def getUserLabels(self):
-    """Collect UserLabels."""
+    """Collect the raw user labels from the path 'nab/labels/raw/'."""
```

Comment:
Right, that's what we want for NAB. But in this class it is not a hardcoded path. getUserLabels looks in self.labelDir